drivers/scheduler/k8s/specs/nginx-sharedv4-longevity/px-nginx-app.yaml

```diff
@@ -0,0 +1,44 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: nginx-service
+spec:
+  selector:
+    app: nginx
+  type: NodePort
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: nginx
+spec:
+  replicas: 24
```

Comment:
instead of hardcoding, let's find a way to parameterize this. Call the parameter as application_replicas_per_namespace. In the helm of the param, mention that this is honored only by statefulsets and apps using read write many PVCs.

Once it's parameterized, remove this new spec and the existing spec itself should be enough.